internal/resourcepolicies/resource_policies.go

```diff
@@ -148,7 +153,43 @@ func (p *Policies) Validate() error {
 	return nil
 }
 
-func GetResourcePoliciesFromConfig(cm *v1.ConfigMap) (*Policies, error) {
+func InitiateResourcePoliciesFromBackup(
```

Comment:
It's trivial but I don't think the "Initiate" in the name makes a lot sense, why not just call it "GetResourcePoliciesFromBackup" or "ResourcePoliciesFromBackup"?

Reply:
I vote "GetResourcePoliciesFromBackup"

Reply:
Renamed to `GetResourcePoliciesFromBackup`.